Example/Tests/AutoRefreshURLProtocolTests.swift

```diff
@@ -90,12 +90,21 @@ class AutoRefreshURLProtocolTests: QuickSpec {
             let refreshUrl = "/oauth/token"
             let successData = "i am not google"
 
-            self.stub(uri(refreshUrl), try! Builders.load(file: "valid-refresh", status: 200))
-
-            self.stub(uri(wantedUrl), Builders.sequentialBuilder([
-                try! Builders.load(file: "empty", status: 401),
-                Builders.load(string: successData, status: 200),
-            ]))
+            let testingNetworkingProxy = TestingNetworkingProxy()
```

Comment:
Do you need this code when you are setting it before each test in the Example/Tests/Configuration.swift ?